src/main/java/org/mule/extension/ftp/internal/source/FtpDirectoryListener.java

```diff
@@ -247,6 +247,10 @@ private boolean processFile(Result<InputStream, FtpFileAttributes> file,
     FtpFileAttributes attributes = file.getAttributes().get();
     String fullPath = attributes.getPath();
 
+    if (LOGGER.isTraceEnabled()) {
+      LOGGER.trace("Processing file {}", attributes);
```

Comment:
ditto

Reply:
The same here. It is what I think we expected it to be. The toString() with all the attributes of the Object "attributes".